tests/tests/test_update_modules.py

```diff
@@ -35,13 +31,7 @@ def test_file_update_module(self, standard_setup_one_docker_client_bootstrapped)
         """Test the file based update module, first with a failed update, then
         a successful one."""
 
-        mender_clients = standard_setup_one_docker_client_bootstrapped.get_mender_clients()
-
-        if not env.host_string:
```

Comment:
Not gonna miss this

Reply:
I will miss all the fun that we had to figure what this was about...

Reply:
I think this should be an interview question hereafter: "What does this piece of code do?"

Reply:
We can make a full quiz just with this test suite :rofl: